quickstarts/microsoft.databricks/databricks-workspace/metadata.json

```diff
@@ -6,5 +6,5 @@
   "summary": "Deploy an Azure Databricks workspace.",
   "githubUsername": "sunainakrishnamoorthy",
```

Comment:
Please update this to your username, generally it is the last updated username and date

Reply:
@bmoore-msft I always took this to mean the original author or the one who "owns" it.  I'm certainly not going to put my name on all of the converted bicep samples, so if sunainakrishnamoorthy doesn't want his there, do you have any other suggestions?

Reply:
the property is *not* the last updated, it's the user who submitted the sample... so you would not change them on update/migration.

This is a doc sample, so you can leave as is (we can't remove it) or see if @mamccrea wants to become the owner.

Reply:
Thanks for the clarification, sorry about the confusion.